donkeycarManagerFront/src/components/register.vue

```diff
@@ -81,31 +66,25 @@ export default {
     },
     async addUser () {
       this.playerTest = await srv.getPlayerByPseudo(this.pseudo)
-      console.log(this.playerTest)
-      console.log(this.playerTest.length)
       if (this.playerTest.length === 0){
-        console.log('pass')
         this.player = await srv.createPlayer(this.pseudo)
-        console.log('pass1.1')
         await srv.addJobs(this.player.player_id)
```

Comment:
This should be outside the condition, in both case you will create the job you have duplicate code.